en/getting_started/flight_modes.md

```diff
@@ -35,7 +36,7 @@ Fixed Wing:
 * Autonomous: [Hold](#hold_fw), [Return](#return_fw), [Mission](#mission_fw), [Takeoff](#takeoff_fw), [Land](#land_fw)
 
 Multicopter: 
-* Manual-Easy: [Position](#position_mc), [Altitude](#altitude_mc), [Manual/Stabilized](#manual_stabilized_mc)
+* Manual-Easy: [Position](#position_mc), [Altitude](#altitude_mc), [Manual/Stabilized](#manual_stabilized_mc), [Orbit](#orbit_mc)
```

Comment:
I think we should also mention that you can use Orbit without any remote? There's no failsafe needed when you loose RC because it's optional.

Reply:
Absolutely. I'll update that now and remove the RC required icon from the top of the page. Thanks.